packages/web-pkg/src/services/passwordPolicy/rules.ts

```diff
@@ -0,0 +1,229 @@
+import { isNaN, isNumber, isObject, isString } from 'lodash-es'
+
+export class MustNotBeEmptyRule {
+  protected $gettext
+
+  constructor({ $gettext }: any) {
```

Comment:
```suggestion
  constructor({ $gettext }: Language) {
```

(Same goes for the other classes down below)